frontend/src/api.js

```diff
@@ -0,0 +1,52 @@
+const API_ENDPOINT = 'http://localhost:4001';
```

Comment:
```suggestion
const API_ENDPOINT = '/api';
```
초반에 언급한 proxy를 이용해서 baseurl을 이런식으로 관리할 수 있도록 작업해보세요!